data/wanderer dire storm.txt

```diff
@@ -0,0 +1,136 @@
+
+ship "Dire Storm"
+	sprite "ship/dire storm"
+	attributes
+		category "Heavy Warship"
+		licenses
+			"Wanderer Military"
+		"cost" 40500000
+		"shields" 54128
+		"hull" 48962
+		"required crew" 72
+		"bunks" 237
+		"mass" 832
+		"drag" 13.9
+		"heat dissipation" 0.5
+		"fuel capacity" 816
+		"cargo space" 424
+		"outfit space" 864
+		"weapon capacity" 276
+		"engine capacity" 240
+		weapon
+			"blast radius" 340
+			"shield damage" 3400
+			"hull damage" 1700
+			"hit force" 5100
+	outfits
+		"Sunbeam" 2
+		"Thunderhead Launcher" 2
+		"Thunderhead Missile" 80
+		"Sunbeam Turret" 2
+		"Wanderer Anti-Missile" 2
+		
+		"Blue Sun Reactor"
+		"Large Biochemical Cell"
+		"Dark Storm Shielding" 2
+		"Bright Cloud Shielding"
+		"Wanderer Ramscoop" 2
+		"Fuel Pod" 2
+		
+		"Type 4 Radiant Thruster" 2
+		"Type 4 Radiant Steering" 2
+		"Hyperdrive"
+		
+	engine -98 160
+	engine 98 160
+	engine 0 289
+	engine -62 232
+	engine 62 232
+	gun -12 -156 "Sunbeam"
+	gun 12 -156 "Sunbeam"
+	gun -17 -134 "Thunderhead Launcher"
+	gun 17 -134 "Thunderhead Launcher"
+	turret -34 16 "Sunbeam Turret"
+	turret 34 16 "Sunbeam Turret"
+	turret -38 105 "Wanderer Anti-Missile"
+	turret 38 105 "Wanderer Anti-Missile"
+	drone 84 63
+	drone -84 63
+	drone -72 63
+	drone 72 63
+	explode "small explosion" 40
+	explode "medium explosion" 70
+	explode "large explosion" 50
+	explode "huge explosion" 20
+	"final explode" "final explosion large" 1
+	description "The Dire Storm is a deep space command, control and science vessel, designed to be one of the first ships through the Eye, to begin cataloging the dangers of the unknown space beyond, and to provide a secure, mobile headquarters for the warships that stay behind to cover the Wanderers' retreat. Endless banks of climate-controlled storage allow the dozen (or so) science labs to take hundreds of soil, water and even biological specimens from desolate worlds. Four drones allow remote, efficient harvesting and in an emergency can provide cover for a fleet of ships."
```

Comment:
"command, control and science vessel" == not a heavy warship
"designed to be one of the first ships through the Eye" == either a fast scouting vessel or a heavy warship

Reply:
> Three more things, in addition to the inline comments added below:
> 
> 1. Why is this in its own file? That would be appropriate for a plug-in, but for content PR'd to the base game, a new ship should go into the existing `wanderer ships.txt` file.
People in discord told me the standard was to have a separate file for this sort of thing. We should maybe agree on the standard for that sort of thing.
> 2. When you submit a PR to add game content, it needs to be complete. You can't just say, "I'm not an artist, someone else can add art." If you're not capable of producing the appropriate art assets, then you probably shouldn't submit the PR until you gain that ability and do it yourself, or arrange ahead of time for someone else to do it so you can submit those assets in the PR.
This is after collaborating on discord with someone for the art. Surveillance/Botspam and I have already done a couple iterations on my original art.
> 3. It's considered a best practice to only only open a PR once everything is done and tested. The review process is optimized for helping you polish up an already solid offering and preventing errors from seeping in; it's not the ideal way to collaboratively generate new content on a continuous basis. The Discord server is a better place to do that.
If you are waiting for content to be perfect and complete before pulling it, that might explain why this project has been in dev hell for two years.

Reply:
It doesn't need to be perfect, but it does need to be _ready to review_. This PR isn't ready to review.